styles/globals.module.scss

```diff
@@ -3,3 +3,48 @@ $nav-height: 70px;
 $footer-height: 180px;
 $mobile-width: 768px;
 $mobile-footer-height: 250px;
+$fadeLeftAnimation: fadeLeft 1s ease-in-out;
+$fadeRightAnimation: fadeRight 1s ease-in-out;
+$fadeUpAnimation: fadeUp 1s ease-in-out;
+$fadeDownAnimation: fadeDown 1s ease-in-out;
+
```

Comment:
I think we should just put all the animations in a separate `animations` module. Thoughts?